packages/terra-content-container/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
    * Added `setFocusOnContainer` prop to provide scroll capability for keyboard only users. (Requires Jest Snapshot updates)
    * Added `borderColor` prop for allowing consumers to change the border color to dark or white depending upon the color of the content container 
 
+* Changed
```

Comment:
this change log should be moved to Unreleased section ( above the released version )

Reply:
Updated